test/data/test_multi_embedding_tensor.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import torch
+
+from torch_frame.data.multi_embedding_tensor import MultiEmbeddingTensor
+
+
+def test_size():
```

Comment:
Looks like you are starting from `values` and `offset`, which is internal storage of multiple embedding tensors.

For testing, can we follow the pattern [here](https://github.com/pyg-team/pytorch-frame/blob/master/test/data/test_multi_nested_tensor.py#L35-L44), where you implement
```python
# Create `tensor_mat: List[Tensor]`. A list of tensors of shape [num_rows, emb_dim] for different columns.
met = MultiEmbeddingTensor.from_tensor_mat(tensor_mat)
```